test/test_database.py

```diff
@@ -0,0 +1,47 @@
+import os
+import unittest
+
+from office_space_allocation import office, livingspace, staff, fellow, amity, db
+
+
+class TestDatabaseOperations(unittest.TestCase):
+    """
+    Tests operations performed when saving or retrieving application state in a 
+    sqlite database
+    """
+
+    def setUp(self):
+        self.amity = amity.Amity()
+
+    def test_can_save_retrieve_application_state_to_sqlite_database(self):
+        """
+        Should be able to save and retrieve all application state to sqlite database
+        """
+        # add rooms
+        rm1 = office.Office("Main Office")
+        rm2 = livingspace.LivingSpace("Living Space")
+        self.amity.add_room(rm1)
+        self.amity.add_room(rm2)
+        # add persons
+        p1 = staff.Staff("Mary", "mary")
```

Comment:
Refer to my previous comment on moving the creation of Persons and Rooms to `setUp()`